src/BrowserKitDriver.php

```diff
@@ -671,6 +671,19 @@ private function getFieldPosition(\DOMElement $fieldNode)
         return 0;
     }
 
+    private function submitFormWith(Form $form, \DOMElement $submitElement)
+    {
+        if ($submitElement->hasAttribute('formaction')) {
+            $form->getNode()->setAttribute('action', $submitElement->getAttribute('formaction'));
```

Comment:
mutating the node is a no-go. It will cause issues as it changes the internal state of the crawler